packages/cozy-stack-client/src/TriggerCollection.spec.js

```diff
@@ -332,6 +351,70 @@ describe('TriggerCollection', () => {
     })
   })
 
+  describe('update', () => {
+    const collection = new TriggerCollection(stackClient)
+
+    beforeEach(() => {
+      stackClient.fetchJSON.mockResolvedValue(UPDATE_RESPONSE_FIXTURE)
+    })
+    afterEach(() => {
+      jest.resetAllMocks()
+    })
+
+    it('should call the right route', async () => {
+      await collection.update({
```

Comment:
you should not use `collection.update`, you should use client.save(). Can you update the tests and the PR please?

Reply:
@Crash-- Since the code and the tests are in cozy-stack-client package, shouldn't I use the collection in the unit tests, and the `collection.save()` here ?

Reply:
Oh my bad. You're right, when unit testing, you can call directly the collection.update(). And it should be named update, not save https://github.com/cozy/cozy-client/blob/master/packages/cozy-client/src/StackLink.js#L126 (I think you didn't see the deletion of my first comment. Next time I'll update it) 

However, in the PR description, you should not mention collection.update(), you should mention client.save(). This is the way you should use it within the caller anyway.

Reply:
Ok I understand now